.gitignore

```diff
@@ -3,5 +3,8 @@
 !.gitignore
 __pycache__
 *.pyc
-*.ckpt
 videos
+resources/coreml/*
+resources/strided_inflated_efficientnet.ckpt
+resources/strided_inflated_mobilenet.ckpt
```

Comment:
How about we create a `resources/backbone` folder? and we ignore all the files in there?